src/modules/github.cpp

```diff
@@ -16,7 +16,14 @@ namespace modules {
    */
   github_module::github_module(const bar_settings& bar, string name_)
       : timer_module<github_module>(bar, move(name_)), m_http(http_util::make_downloader()) {
+    using namespace std::string_literals;
```

Comment:
I think this is already included via `common.hpp`

Reply:
Yeah indeed, I'll remove that.